.dockerignore

```diff
@@ -0,0 +1,6 @@
+.git
+.github
+.ci
+.idea
+.vscode
+build
```

Comment:
I a not sure why we need this file at all.

Reply:
we don't need that file anymore. I used it when I was trying to speed up the docker build phase but since we moved the Dockerfile it is not needed anymore. I'll delete that file